test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -861,6 +862,10 @@ TEST_P(ProxyProtocolTest, V1TooLongWithAllowNoProxyProtocol) {
 }
 
 TEST_P(ProxyProtocolTest, V2ParseExtensions) {
+  TestScopedRuntime scoped_runtime;
+  scoped_runtime.mergeValues({
+      {"envoy.reloadable_features.use_typed_metadata_in_proxy_protocol_listener", "false"},
```

Comment:
Are these still needed? Here and several more tests. I'm guessing this is leftover from when you were not emitting the old data when the flag was true.

Reply:
Should we leave it in few places for coverage sake? E.g. to exercise code path when user disables the flag.

Reply:
Left it in few places with explicit check that typed metadata is not populated when flag is set to false.